assets/sass/patterns/atoms/block-link.scss

```diff
@@ -116,9 +120,9 @@
     bottom: 0;
     left: 0;
     z-index: -1;
-    background-image: linear-gradient(to top, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5));
+    background-color: rgba(0, 0, 0, 0.5);
     .block-link:hover & {
-      background-image: linear-gradient(to top, rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5));
+      background-color: rgba(0, 0, 0, 0.8);
```

Comment:
Use sass variable `$color-overlay` for this, so: `background-color: $color-overlay;`